frontend/src/Components/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const HomePage = () => {
```

Comment:
It is not a must have, but cleaner to use arrow function property to omit return key word. 
`const Home = () => (
<div>....</div>
)`

Reply:
Yeah i understand, this is just a placeholder component for now.